### DISCUSSION TITLE
ComboBox styling <-> Option Menu

### DISCUSSION THREAD

**Discussion by kfsone** [TomSchimansky/CustomTkinter#784] [General]
I'm not crazy about the Option Menu looking like a button

![image](https://user-images.githubusercontent.com/323009/204935249-27831f47-5dfe-4418-a1ae-6299ac8b5da1.png)

I get why you might, but to me the giant drop-down button is a big enough hint that a color change is uncalled for.

The appearance I want is that of the ComboBox without the ability to change the text.

![image](https://user-images.githubusercontent.com/323009/204935419-254ea7dd-5378-4ecb-9e14-314d37bddf8b.png)

**Comment by TomSchimansky** [comment]
The optionemenu has no border option, but you can change the colors how you like it. You can make a local copy of the blue.json theme file and change the values for the optionmenu and the load this theme file.
